server.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+
+const recipeRouter = require("./recipes/recipe-router.js");
+const ingredientRouter = require("./ingredients/ingredient-router.js");
+
+const server = express();
+
+server.use(express.json());
+server.use("/api/recipes", recipeRouter);
+server.use("/api/ingredients", ingredientRouter);
+
+server.get("/", (req, res) => {
+  res.send("Server is working!");
+});
+
+module.exports = server;
```

Comment:
overall i would suggest adding much more comments to your code, but it's excellent work